lib/Handler.js

```diff
@@ -8,17 +8,21 @@
 
 const request = require("./Request");
 
+const API_BASE_URL = "https://sayonika.maidcafe.cf/api/v1/";
+
 /**
  * Main class that handles Queries to Sayonika
  */
 module.exports = class {
     /**
-     * 
+     *
      * @param {String} token your API Authorization.
-     * @see {link} https://sayonika.maidcafe.cf/developers 
+     * @param {String} api_base_url the base URL for api calls
+     * @see {link} https://sayonika.maidcafe.cf/developers
      */
-    constructor(token) {
+    constructor(token, api_base_url=API_BASE_URL) {
```

Comment:
instead of a `const` on the top level code, just put it where you define it in the parameter